engine/CMakeLists.txt

```diff
@@ -1181,17 +1190,16 @@ ELSEIF (XCODE)
     #  See https://cmake.org/cmake/help/v3.3/module/CPackDMG.html
     # Bundle -> Compressed Disk Image
     # PackageMaker - see https://cmake.org/cmake/help/v3.3/module/CPackPackageMaker.html
-    SET(CPACK_GENERATOR "DragNDrop" "BUNDLE" "TBZ2" "TGZ" "TXZ")
+    SET(CPACK_GENERATOR "DragNDrop" "BUNDLE" "TXZ")
 ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
     MESSAGE("-- Configuring Packaging for Linux")
-    SET(CPACK_GENERATOR "TBZ2" "TGZ" "TXZ")
-    # Linux
+    SET(CPACK_GENERATOR "TXZ")
```

Comment:
I'd prefer to keep TBZ2 on Linux as well. TGZ can get dropped since GH provides that.

Reply:
Fixed